src/stacks-hierarchy/Root_PurchaseConfirmationScreen/Root_PurchaseConfirmationScreen.tsx

```diff
@@ -304,7 +304,7 @@ export const Root_PurchaseConfirmationScreen: React.FC<Props> = ({
             style={{margin: theme.spacing.medium}}
           />
         ) : (
-          <View>
+          <View style={{marginBottom: theme.spacing.small}}>
```

Comment:
I don't think this is needed after the `container` view was added. That adds some padding all around